server/graphql/v2/interface/IsMemberOf.js

```diff
@@ -59,6 +61,10 @@ export const IsMemberOfFields = {
           [Op.in]: args.accountType.map(value => AccountTypeToModelMapping[value]),
         };
       }
+      if (args.account) {
+        const account = await fetchAccountWithReference(args.account);
```

Comment:
Small perf improvement

```suggestion
        const account = await fetchAccountWithReference(args.account, { loaders: req.loaders });
```